src/main/webapp/receipt-analysis.html

```diff
@@ -23,6 +23,16 @@
 <title>Receipt Analysis</title>
 
 <body onload="load()">
+  <button
+    id="home-button"
+    href="/"
```

Comment:
Why is there an `href` here but not in upload.html?

Reply:
Removed.